app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesActivity.java

```diff
@@ -46,6 +50,25 @@ public void prepareView() {
         extractBundle();
     }
 
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.settings:
+                Intent intent = new Intent(this, SettingsActivity.class);
```

Comment:
please create intent from internal static method in SettingsActivity class (like EditActivity class)